src/components/MailingList/MailingList.jsx

```diff
@@ -10,65 +10,63 @@ const EmailForm = styled.form`
 `;
 const EmailInputBox = styled.input`
     min-width: 250px;
-    border: #555555 solid 2px;
+    border: 2px solid #555;
+    border-right: 0;
+    height: 50px;
+    border-top-left-radius: 16px;
+    border-bottom-left-radius: 16px;
+    padding: 0 16px;
 `;
 const EmailSubmitButton = styled.input`
-    background-color: #f15d24;
     color: white;
-    border: none;
+    border: 2px solid #555;
+    border-left: 0;
     text-decoration: none;
     padding-top: 3px;
     padding-bottom: 3px;
     cursor: pointer;
-    min-width: 50px;
-`;
-
-class MailingList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = { address: "", submitted: false };
-
-        this.handleChange = this.handleChange.bind(this);
-        this.addEmailToList = this.addEmailToList.bind(this);
+    background: rgb(239, 133, 62);
+    width: 60px;
+    height: 50px;
+    border-top-right-radius: 16px;
+    border-bottom-right-radius: 16px;
+    &:hover {
+      background: rgb(222, 103, 63);
     }
+`;
 
-    addEmailToList(e) {
-        e.preventDefault();
+export default function MailingList() {
+  const [address, setAddress] = useState("");
+  const [submitted, setSubmitted] = useState(false);
 
-        const payload = {
-            Email: this.state.address,
-        };
-        /**
-         * TODO: refactor when merging with attendance branch, which adds better email error handling.
-         */
-        addEmailFetch(payload);
+  function addEmailToList(e) {
+    e.preventDefault();
 
-        this.setState({
-            address: "",
-            submitted: true,
-        });
-    }
+    const payload = {
+      Email: address,
+    };
+    /**
+     * TODO: refactor when merging with attendance branch, which adds better email error handling.
+     */
+    addEmailFetch(payload);
 
-    handleChange(event) {
-        this.setState({ address: event.target.value });
-    }
+    setAddress("");
+    setSubmitted(true);
```

Comment:
Clear these only if the response is okay. Backend is still down so it will be broken.